.github/workflows/ci.yaml

```diff
@@ -166,6 +171,11 @@ jobs:
           CUDA_VER: "11.6.0"
           CENTOS_VER: "8"
 
+        - DOCKERIMAGE: linux-anvil-ppc64le-cuda
+          DOCKERTAG: "11.7"
+          CUDA_VER: "11.7.0"
+          CENTOS_VER: "8"
```

Comment:
We may need to change this to use some other image (probably UBI) as CentOS 8 has been dropped

Reply:
Cool. I can tweak the template and variables accordingly, so that this is configurable instead of hardcoded.

Reply:
Sure let's give that try. AIUI UBI is pretty close to RHEL so it should work well for us. Though will be interesting to see if any differences pop up

Reply:
From what I can see on [DockerHub](https://hub.docker.com/r/nvidia/cuda/tags?page=1&name=11.7.0-devel-), UBI is the only image with ppc64le support. Seems like the right choice.

Reply:
@jakirkham For the moment, I only replaced the image names in `linux-anvil-ppc64le-cuda/Dockerfile` and `linux-anvil-aarch64-cuda/Dockerfile`. I don't think replacing the variable name `CENTOS_VER` with something more generic would be very easy, given its prevalence in this repo.

Reply:
Note: it looks like `ubi8` images are available for ppc64le and aarch64 for CUDA versions 11.0.3 and newer.

Reply:
Think if `CENTOS_VER` is `8` that should be close enough to what we need (even though it isn't really CentOS)

Right we could use this for the other images as well. Maybe we should hold off on that for now